lib/ansible/modules/database/postgresql/postgresql_privs.py

```diff
@@ -265,6 +271,35 @@
     type: default_privs
     role: reader
 
+# Available since version 2.8
+# ALTER DEFAULT PRIVILEGES FOR ROLE librarian IN SCHEMA library GRANT SELECT ON TABLES TO reader
+# GRANT SELECT privileges for new TABLES objects created by librarian as
+# default to the role reader.
+# For specific
+- postgresql_privs:
+    db: library
+    schema: library
+    objs: TABLES
+    privs: SELECT
+    type: default_privs
+    role: reader
+    target_roles: librarian
+
+# Available since version 2.8
+# ALTER DEFAULT PRIVILEGES FOR ROLE librarian IN SCHEMA library REVOKE SELECT ON TABLES FROM reader
+# REVOKE SELECT privileges for new TABLES objects created by librarian as
+# default from the role reader.
+# For specific
+- postgresql_privs:
+    db: library
+    state: absent
+    schema: library
+    objs: TABLES
+    privs: SELECT
+    type: default_privs
+    role: reader
+    target_roles: librarian
+
 """
 
 import traceback
```

Comment:
Traceback should not be needed, Ansible adds traceback information on errors in except-blocks automatically.

Reply:
@tcraxs these formatting fixes are all that remain. then merge, I hope. There are some issues accumulated for postgresql_privs that I want to start to solve after the merge of this PR. So, please, don't postpone it ;)

Reply:
I can start now but, I'm afraid, there will be many merge conflicts with your PR that you / or I need to solve.

Reply:
Sorry I was not able to get some free time for this. I will resolve the merge conflicts right now.
Regarding the "import traceback", I think that is out of my scope. I'm not sure how to resolve that, because it was already in the code before my change. I think we have to rewrite a lot of the code to remove the traceback-thing and port it to match it with the Ansible default error-handling.